src/main/java/de/caritas/cob/agencyservice/api/admin/service/UserAdminService.java

```diff
@@ -88,8 +88,6 @@ public List<ConsultantAdminResponseDTO> getConsultantsOfAgency(Long agencyId, in
     sortBy.setField(FieldEnum.LASTNAME);
     sortBy.setOrder(OrderEnum.ASC);
 
-    return controllerApi
-        .getConsultants(currentPage, perPage, consultantFilter, sortBy)
-        .getEmbedded();
+    return controllerApi.getAgencyConsultants(String.valueOf(agencyId)).getEmbedded();
```

Comment:
hmm is this method only used for the usecase of deletion?
Because you are changing the logic here for sure. If it's only used for deletion, then we should also remove currentPage, perPage and filter... also in the signature of the method.

Currently, what you are changing here is really far from clean code :/